compiler/damlc/daml-stdlib-src/DA/Internal/Interface.daml

```diff
@@ -175,3 +179,30 @@ class HasInterfaceView i v | i -> v where
 
 view : forall i v. HasInterfaceView i v => i -> v
 view = _view
+
+class HasFromAnyView i v | i -> v where
+  -- | HIDE
+  _fromAnyView : proxy i -> Any -> Optional v
+
+class HasToAnyView i v | i -> v where
+  -- | HIDE
+  _toAnyView : proxy i -> v -> Any
```

Comment:
As discussed last week, we don't expect client code to construct views, so this class is unnecessary

Reply:
done